pkg/shell/nav.jsx

```diff
@@ -139,17 +139,17 @@ export class CockpitNav extends React.Component {
                 <SearchInput placeholder={_("Search")} value={this.state.search} onChange={(_, search) => this.setState({ search })} onClear={() => this.setState({ search: "" })} className="search" />
                 <Nav onSelect={this.onSelect} theme="dark">
                     { groups.map(g =>
-                        <section className="pf-c-nav__section" aria-labelledby={"section-title-" + g.name} key={g.name}>
+                        <section className="pf-v5-c-nav__section" aria-labelledby={"section-title-" + g.name} key={g.name}>
                             <div className="nav-group-heading">
-                                <h2 className="pf-c-nav__section-title" id={"section-title-" + g.name}>{g.name}</h2>
+                                <h2 className="pf-v5-c-nav__section-title" id={"section-title-" + g.name}>{g.name}</h2>
                                 { g.action &&
-                                    <a className="pf-c-nav__section-title nav-item" href={g.action.path} onClick={ ev => {
+                                    <a className="pf-v5-c-nav__section-title nav-item" href={g.action.path} onClick={ ev => {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18806-20230519-161212-54b4f0d5-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)